### PR TITLE
Adapt comp image tests

### PR DESCRIPTION
**Pull request by nilseling**
I noticed that I never managed to induce spillover in images in a correct way for the unit tests. This is now more elegant and checks the spillover correction correctly.

### CODE REVIEW

File: tests/testthat/test_compImage.R
Comment (lassedochreden) [inline tests/testthat/test_compImage.R:1]: At first glimpse, this looks good to me @nilseling!

**Review by lassedochreden — APPROVED** [review]
Hi @nilseling, this looks good and well tested to me now. Yet R CMD Check is failing right now - Will leave a comment on that in the conversation.

**Comment by lassedochreden** [comment]
Hi @nilseling, as you can see R CMD Check is failing due to **Cache PhantomJS** (Error: Input required and not supplied: path). Furthermore, there are two deprecation warnings from _set-output_ and _node12_. 

Can you comment on that and potentially fix it? 

Thanks in advance for clarifying!

**Comment by nilseling** [comment]
So the build issue on devel is due to a bunch of packages not being available at the moment. I'm not fully sure why this is but I think we can ignore this for now.

**Pull request merged**